### PR TITLE
DOC update README

### PR DESCRIPTION
**Pull request by tomMoral**
update the README with description of the task.

Can you review @bruAristimunha @chris-mrn ?

### CODE REVIEW

File: README.rst
Comment (chris-mrn) [inline README.rst:1]: I find it really nice! Maybe we could just add one phrase or two to talk about data augmentation

Comment (bruAristimunha) [reply README.rst:1]: I think we can do that later, given that we're still changing things around for the augmentation part. What do you think @chris-mrn?

**Review by bruAristimunha — APPROVED** [review]

**Comment by bruAristimunha** [comment]
LGTM :) If @chris-mrn was ok with it, we can merge. I leave merge button with you, @tomMoral and @chris-mrn :)

File: README.rst
Comment (tomMoral) [reply README.rst:1]: agreed! lt's merge this one them :)

**Pull request merged**
